mopro-cli/README.md

```diff
@@ -66,11 +67,23 @@ mopro build --platforms ios
 mopro test
 ```
 
+### Android
+
+Initialize and build an app with Android support.
+> Currently support [Android Studio](https://developer.android.com/studio)
+
+```
+mopro init --platforms android
+cd mopro-example-app
+mopro prepare
+mopro build --platforms android
+```
+
 ### Exporting bindings
 
 To export bindings to a different directory:
 
-`mopro export-bindings --destination <DESTINATION_DIR>`
+`mopro export-bindings --destination <DESTINATION_DIR> --platforms <IOS_OR_ANDROID>`
```

Comment:
it works as previous platforms
it can be one platform or multiple platforms (`--platforms ios android`)
but I improved it a little bit here: [8d57287](https://github.com/oskarth/mopro/pull/103/commits/8d57287bfe49eb6b7b8832957073beef528a2915)

Reply:
Oh right, I was confused because the comment said IOS_OR_ANDROID